crates/router/src/core/payments/operations/payment_confirm.rs

```diff
@@ -427,6 +437,10 @@ impl<F: Clone> UpdateTracker<F, PaymentData<F>, api::PaymentsRequest> for Paymen
                     return_url,
                     business_country,
                     business_label,
+                    description,
+                    statement_descriptor_name,
+                    statement_descriptor_suffix,
+                    client_secret,
```

Comment:
This need not be updated, as this field is generated internally during `PaymentsCreate`.